tools/tidy/tests/RegisterHasNoResetLoopTest.cpp

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: Michael Popoloski
+// SPDX-License-Identifier: MIT
+
+#include "Test.h"
+#include "TidyFactory.h"
+
+TEST_CASE("RegisterHasNoReset: For loop iteration variable should not trigger warning") {
```

Comment:
Please add this test case to the existing RegisterHasNoResetTest.cpp file instead.

Reply:
Please rebase the PR on top of master. Make sure pre-commit has run on the changed files. Make sure the build actually succeeds (wait longer than 120s if you have to) and that the new test actually runs and passes.